### DISCUSSION TITLE
Deploying keystoneJS to Fly.io

### DISCUSSION THREAD

**Discussion by muco-rolle** [keystonejs/keystone#8043] [Questions]
Hello everyone,

Is there any guide on how to deploy KeystoneJS to fly.io

**Comment by flexdinesh** [comment]
We don't have one yet @muco-rolle. We're looking at writing more guides around deploying Keystone to various hosting providers and fly.io is on the list. If you have already figured out how to deploy Keystone to fly.io please let us know how the process looked like and we could include it in the guide.
